[Serializer][Translation] Deprecate passing a non-empty CSV escape char (#57827)
symfony/symfony: src/Symfony/Component/Serializer/CHANGELOG.md

CHANGELOG
=========

7.2
---

>  * Deprecate passing a non-empty escape character to the `csv_escape_char` option of `CsvEncoder` and the `CsvEncoder::ESCAPE_CHAR_KEY` constant

Reviewer: ```suggestion
 * Deprecate the `csv_escape_char` context option of `CsvEncoder` and the `CsvEncoder::ESCAPE_CHAR_KEY` constant
```

Same for `UPGRADE-7.2.md`

Author: Updated :+1:

Reviewer: I think you rolled back these changes in the last commit

Author: I don't know what happened, but it should be good now!
